### PR TITLE
Added generic DAO interface without executor for all entities DAO imp…

### PR DESCRIPTION
**Pull request by vsrover**
…lementation and cleared Customers entity for next updating

### CODE REVIEW

File: src/main/java/dao/CustomerDAOImpl.java [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+import java.sql.*;
+import java.util.HashSet;
+import java.util.Set;
+
+public class CustomerDAOImpl implements DAO<Customer>{
+
+    private Customer extractCustomerFromResultSet(ResultSet rs) throws SQLException {
+        Customer customer = new Customer();
+        customer.setCustomerId(rs.getInt("idCustomer"));
+        customer.setAdmin(rs.getBoolean("isAdmin"));
```
Comment (SMishAlex) [inline src/main/java/dao/CustomerDAOImpl.java]: customer.isAdmin? sounds fun. we don't have admin role so you can just remove it

File: src/main/java/entities/Customer.java [hunk 2]
```diff
@@ -1,11 +1,24 @@
 package entities;
 
 public class Customer {
+
     private int customerId;
     private boolean isAdmin;
     private String customerName;
-    private String LastName;
-    private String personId;
+    private String customerLastName;
```
Comment (SMishAlex) [inline src/main/java/entities/Customer.java]: we can have just name or first and last name

File: src/main/java/entities/Customer.java [hunk 3]
```diff
@@ -1,11 +1,24 @@
...
+    private String customerPersonId;
```
Comment (SMishAlex) [inline src/main/java/entities/Customer.java]: what is personId and how will we use it?

File: src/main/java/services/ConnectionFactory.java [hunk 4]
```diff
@@ -10,9 +10,9 @@
  * @author klysov
  */
 public class ConnectionFactory {
-    private static final String URL = "jdbc:mysql://localhost:3306/ticketsystem";
-    private static final String USER = "testuser";
-    private static final String PASS = "testpassword";
+    private static final String URL = "jdbc:mysql://localhost:3306/plane";
+    private static final String USER = "root";
+    private static final String PASS = "root";
```
Comment (SMishAlex) [inline src/main/java/services/ConnectionFactory.java]: you have to have same database properties such as URL, USER, PASSWORD
it should not be changed in every commit

File: src/main/java/dao/DAO.java [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+import java.sql.SQLException;
+import java.util.Set;
+
+public interface DAO<T>{
+    Set<T> getAll() throws SQLException;
+    T getById(int id) throws SQLException;
+    boolean insert(T t) throws SQLException;
+    boolean update(T t) throws SQLException;
+    boolean delete(T t) throws SQLException;
+}
```
Comment (SMishAlex) [inline src/main/java/dao/DAO.java:12]: good job

File: src/main/java/dao/CustomerDAOImpl.java (hunk 1 above)
Comment (vsrover) [reply src/main/java/dao/CustomerDAOImpl.java]: Okey, I will remove this one

File: src/main/java/entities/Customer.java (hunk 2 above)
Comment (vsrover) [reply src/main/java/entities/Customer.java]: Ok, understand

File: src/main/java/entities/Customer.java (hunk 3 above)
Comment (vsrover) [reply src/main/java/entities/Customer.java]: PersonID - this is the passport data

File: src/main/java/services/ConnectionFactory.java (hunk 4 above)
Comment (vsrover) [reply src/main/java/services/ConnectionFactory.java]: I don't know any idea how solve this issue

File: src/main/java/dao/DAO.java (hunk 5 above)
Comment (vsrover) [reply src/main/java/dao/DAO.java:12]: Thnanks

File: src/main/java/entities/Customer.java (hunk 3 above)
Comment (SMishAlex) [reply src/main/java/entities/Customer.java]: ok then

File: src/main/java/services/ConnectionFactory.java (hunk 4 above)
Comment (SMishAlex) [reply src/main/java/services/ConnectionFactory.java]: all of you just configure your dbs in the same way

File: src/main/java/entities/Customer.java (hunk 3 above)
Comment (vsrover) [reply src/main/java/entities/Customer.java]: such as 40 08 131-33-4444

File: src/main/java/services/ConnectionFactory.java (hunk 4 above)
Comment (vsrover) [reply src/main/java/services/ConnectionFactory.java]: Done

**Pull request merged**